teachopencadd/talktorials/018_md_simulation/talktorial.ipynb

```diff
@@ -0,0 +1,1317 @@
+{
```

Comment:
<p>typo: <strong>resiudes</strong> --&gt; <strong>residues</strong></p><p>There is a python convention that favors imperative docstrings (<a href="https://stackoverflow.com/questions/59902102/why-is-imperative-mood-important-for-docstrings" target="_blank">stackoverflow</a>), i.e.,: <strong>Use</strong> instead of <strong>Uses</strong>.</p><p>Could you specify by a keyword with default true, that missing residues should no be built and add this to the docstring?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>